frontend/analytics/plausible.ts

```diff
@@ -0,0 +1,8 @@
+const trackEvent = ({ eventName }: { eventName: string }) => {
```

Comment:
I reckon the `eventName` type could be also forced to be from the enum so that later developers remember to use the enum.